tests/integration/secrets/auth_example.json

```diff
@@ -2,29 +2,47 @@
     "stackData": {
         "name": "<YOUR STACK NAME>",
         "url": "<YOUR STACK URL>",
+        "wss": "<YOUR STACK WSS URL>",
```

Comment:
what is a "WSS URL" ?

Reply:
It's the url that looks like this: `wss://loadbalancer-api-staging-04.highfidelity.io:8001/` I don't know what to call it.